config/riff-http-gateway.yaml

```diff
@@ -29,6 +29,25 @@ subjects:
   name: default
   namespace: riff-system
 ---
+apiVersion: v1
```

Comment:
This should be put in a separate file. Sometimes we want a ClusterIP service, sometimes a LoadBalancer, etc...

Reply:
We could look into parameterizing the service type during installation. (may-be from helm). I think for now, ClusterIP should work.